resources/integration_test/tests/query.yaml

```diff
@@ -188,13 +188,15 @@ tests:
     sql: SELECT * FROM types_other
     expected_result_extended_json: [ {
       "0": 0,
-      "1": [1,2,3],
+      "1": [{"$numberInt": "1"}, {"$numberInt": "2"}, {"$numberInt": "3"},
```

Comment:
Just a suggestion: Maybe we could use the relaxed extended JSON format. This would help make the data a little less verbose and easier to display in a UI based tool?

Reply:
Switched to `JsonMode.RELAXED`